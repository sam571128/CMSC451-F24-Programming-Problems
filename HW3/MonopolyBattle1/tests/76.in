55 1732 348295192
53 30
54 13
47 32
50 8
26 14
16 25
2 36
16 54
28 39
36 31
31 47
41 27
36 25
44 16
37 11
46 33
24 38
18 43
40 41
48 7
17 9
36 28
14 26
10 44
36 9
12 38
45 15
40 10
2 37
19 52
9 10
8 4
33 6
27 18
20 24
26 30
47 18
14 17
10 33
9 42
43 40
7 20
41 16
37 40
48 31
22 37
12 12
29 45
29 37
52 31
46 3
12 48
6 36
50 17
1 20
33 9
11 42
14 43
29 53
30 3
13 40
10 2
11 3
38 42
31 42
40 55
37 37
15 29
10 9
53 27
10 4
42 7
1 52
16 37
50 1
22 32
15 21
49 8
46 16
33 19
43 51
32 28
20 17
39 50
47 54
3 41
31 25
23 21
21 11
4 8
52 17
44 25
17 18
9 54
14 5
37 35
44 46
15 35
46 46
1 38
24 21
16 28
55 46
53 40
28 55
18 9
25 15
53 9
34 5
17 6
17 54
47 31
45 55
35 33
25 14
53 17
50 51
1 26
12 47
34 23
8 18
25 21
33 24
19 22
31 11
53 23
27 27
14 21
48 34
41 36
31 10
1 39
30 8
42 54
23 41
36 7
18 32
49 35
6 3
11 5
53 52
29 15
47 43
53 41
6 17
48 38
46 5
13 14
16 55
1 19
15 17
10 49
39 43
52 32
42 11
3 7
22 33
7 47
27 50
22 31
6 51
34 28
18 22
7 52
29 21
8 1
39 3
6 53
13 28
28 4
7 29
47 51
45 39
35 22
36 34
32 50
11 9
55 12
15 4
48 20
29 34
13 32
28 52
19 9
12 26
10 51
2 5
45 5
40 33
55 26
7 28
32 36
20 19
27 41
18 51
11 19
9 33
14 35
43 41
33 13
19 29
10 7
17 12
46 44
31 29
21 55
24 9
14 52
29 40
50 26
15 25
45 49
50 30
25 23
37 29
13 23
8 49
35 31
25 43
38 23
23 8
8 45
41 17
27 23
50 2
30 2
43 46
9 43
18 36
12 45
49 7
13 3
18 20
4 32
6 32
33 29
51 31
36 51
52 1
24 40
27 55
35 41
31 14
21 46
24 43
14 28
11 16
25 20
41 34
45 37
28 41
4 14
15 55
38 17
33 12
48 37
37 30
28 23
23 32
19 25
29 17
1 54
45 7
42 46
31 28
46 19
15 40
52 13
43 27
55 2
51 17
3 30
11 2
21 6
53 16
2 38
45 38
32 21
7 25
29 2
17 20
50 35
34 21
54 33
39 19
12 4
14 55
5 44
9 51
9 46
8 7
12 14
36 46
27 21
5 4
31 44
17 19
31 22
6 49
47 20
52 35
41 9
25 2
16 26
50 29
26 44
1 37
19 28
3 17
13 47
43 53
2 16
29 23
37 2
16 40
32 33
53 26
35 6
21 26
23 3
43 52
12 19
40 12
47 37
5 22
6 39
31 38
10 32
25 1
30 34
2 39
54 39
14 19
15 5
13 24
40 42
1 6
46 23
52 18
49 30
9 27
6 33
44 44
15 34
14 34
6 55
25 28
50 24
32 22
37 53
52 36
19 32
17 3
46 29
22 46
17 2
53 24
48 13
28 29
47 5
42 26
28 33
32 43
47 14
21 8
11 18
41 4
1 29
29 49
26 49
55 13
8 48
40 19
10 53
40 20
21 35
39 51
23 22
44 18
20 13
37 20
23 53
21 47
6 14
55 31
43 49
16 27
46 45
8 5
15 28
46 49
22 53
46 12
34 9
36 3
21 52
41 11
40 4
22 1
8 42
3 18
11 22
22 17
4 2
49 12
37 42
48 11
17 47
54 19
3 34
5 21
41 33
35 30
35 35
13 9
12 39
50 46
6 38
55 34
14 29
53 43
34 7
14 18
10 17
16 22
24 49
18 7
4 12
22 51
26 22
15 22
15 37
22 23
1 47
44 1
3 24
36 33
48 8
29 18
46 34
27 29
20 31
45 42
26 38
20 18
14 38
52 20
31 17
53 38
8 14
28 53
49 2
30 14
46 22
39 35
10 3
15 42
23 36
40 52
30 50
39 41
1 25
35 40
8 11
28 22
26 6
3 4
35 16
5 34
44 34
16 12
14 39
13 30
20 55
28 20
10 6
13 13
52 26
34 31
9 35
13 29
18 21
18 34
28 21
18 37
31 23
10 41
19 2
5 29
25 52
20 11
46 24
54 41
53 55
28 25
6 28
45 40
43 8
44 37
54 38
33 1
13 6
46 55
38 30
31 2
53 8
6 40
51 24
6 7
11 32
44 6
28 34
20 20
52 47
16 41
8 22
38 4
53 19
37 24
14 15
8 46
38 44
19 41
54 25
39 26
7 13
54 50
24 15
37 55
7 45
52 22
7 12
21 50
23 27
34 47
51 34
44 52
3 50
40 7
49 37
7 21
30 30
10 5
51 5
20 37
39 40
7 31
55 16
30 27
22 45
3 42
7 8
11 37
41 20
43 45
13 19
10 28
26 39
14 1
15 41
55 32
6 34
50 20
39 5
4 43
51 41
28 14
20 39
33 37
4 25
55 50
1 33
37 48
4 50
33 26
48 23
42 37
54 55
32 20
36 21
2 49
29 8
46 32
26 21
26 10
18 5
27 51
10 26
31 7
14 16
3 9
21 1
21 33
44 48
5 42
37 45
48 46
44 9
7 26
37 21
47 22
53 11
16 2
22 29
52 44
11 27
12 6
43 32
14 50
26 33
45 44
47 3
40 28
8 37
46 7
51 4
18 25
45 35
16 30
3 32
40 36
39 30
30 48
54 32
52 3
36 13
16 18
4 11
36 16
42 17
35 25
55 52
25 40
54 40
38 12
30 19
1 7
36 50
39 8
37 16
9 32
44 3
34 2
13 39
20 47
22 6
9 53
3 38
1 45
10 22
43 30
40 5
2 48
47 25
25 55
47 39
14 31
6 15
35 14
43 7
3 19
19 35
5 41
52 7
23 50
41 55
9 36
41 37
28 50
36 53
45 14
13 54
40 46
13 15
15 2
3 47
35 19
5 18
35 26
48 22
45 19
38 28
29 13
38 37
15 26
12 37
51 15
47 38
21 22
29 36
43 10
25 45
30 44
28 48
34 8
39 45
11 53
8 10
24 2
7 53
13 26
25 12
30 16
17 13
47 52
52 23
47 17
4 42
39 6
41 45
6 37
17 51
45 1
1 30
40 25
26 25
4 19
33 7
2 32
37 17
31 46
37 39
10 13
32 6
43 48
19 12
25 9
15 51
27 31
24 3
48 51
43 17
29 7
37 34
9 38
18 2
26 52
40 30
46 2
16 23
43 54
45 4
25 6
31 49
34 46
1 22
36 37
16 39
38 9
32 40
7 7
17 16
23 48
7 37
31 41
43 1
49 27
43 16
30 7
50 32
30 53
29 25
34 30
44 12
32 47
50 49
4 28
25 51
32 8
8 44
28 51
21 32
52 29
32 18
9 4
47 8
27 10
33 33
28 24
41 42
42 18
35 21
48 18
24 44
24 14
45 25
20 8
47 30
8 6
27 15
2 52
26 17
11 6
21 17
13 49
23 42
30 23
39 14
25 7
20 32
48 49
36 55
10 14
51 53
30 41
7 36
51 10
9 45
17 25
1 40
41 51
9 50
27 14
34 35
48 40
52 41
33 32
37 15
21 2
5 30
43 9
35 9
11 40
48 28
29 1
53 6
49 18
55 33
8 28
18 55
51 40
3 8
29 33
53 5
20 49
13 11
17 40
14 44
37 25
20 26
31 48
6 29
16 1
53 3
29 22
32 26
39 33
25 26
24 55
20 36
14 9
54 23
19 13
15 54
2 43
43 15
7 35
26 12
39 28
55 6
43 11
10 48
10 23
24 51
39 44
1 43
25 5
10 47
17 35
11 10
11 43
19 26
9 52
38 39
50 41
17 30
40 23
8 2
25 34
37 28
25 8
26 28
22 19
11 21
42 28
30 45
8 38
50 14
2 28
38 6
28 31
42 19
5 40
16 10
13 18
2 41
21 36
33 23
13 51
28 12
4 18
53 1
14 40
33 41
16 43
25 22
55 8
49 6
55 11
34 18
10 46
35 13
16 9
49 16
35 34
24 20
24 35
41 46
21 38
14 53
41 52
47 27
27 8
43 22
39 9
40 54
12 46
20 50
21 42
9 47
30 55
4 46
16 49
8 31
28 9
10 54
53 21
54 29
43 20
8 54
40 45
53 54
23 49
39 23
53 51
53 31
9 7
41 3
34 33
13 1
7 51
23 37
10 21
22 52
26 24
5 14
27 1
41 48
1 50
36 38
26 13
27 11
49 34
19 45
40 21
48 48
25 19
16 19
10 10
55 20
7 54
28 7
42 4
27 47
11 26
50 27
15 3
9 28
24 16
36 19
14 27
8 9
2 22
22 24
55 19
55 40
38 11
28 2
34 41
9 20
30 13
52 28
26 31
39 11
15 39
18 3
17 10
48 14
44 23
20 52
19 43
31 52
10 8
53 53
2 42
34 4
9 21
53 28
48 24
43 34
26 41
23 14
13 37
52 49
41 19
42 42
12 43
9 34
22 49
12 42
9 12
39 52
52 54
31 12
53 48
11 52
41 40
31 20
41 15
40 32
48 32
41 38
52 14
20 44
47 10
51 37
9 3
27 17
40 22
14 46
11 17
3 26
6 54
7 15
12 11
39 37
5 37
8 12
31 53
12 18
33 11
28 40
27 7
53 25
48 35
35 51
23 34
14 47
2 50
5 45
40 16
25 11
46 17
39 2
20 27
7 41
30 9
37 36
13 48
23 33
14 22
30 51
45 23
49 45
2 25
53 4
22 40
4 37
13 10
43 4
31 55
49 47
52 45
39 7
31 43
32 46
14 45
55 49
40 31
1 53
36 43
22 13
29 5
13 21
2 34
38 22
4 5
28 26
2 10
55 22
35 5
12 35
9 19
46 21
42 38
32 15
17 37
24 1
43 28
23 29
52 51
46 8
38 7
49 23
6 45
24 7
4 44
54 3
50 40
36 14
30 5
9 44
2 20
39 32
20 43
29 28
39 10
45 52
32 49
39 16
16 50
6 27
53 32
47 6
23 10
29 19
25 38
30 35
52 52
26 55
51 28
18 38
18 31
50 22
5 48
29 20
44 50
48 25
12 49
47 7
10 42
14 42
47 19
36 45
53 49
55 45
7 14
52 50
1 34
48 4
42 32
26 32
38 2
21 21
11 45
6 23
54 36
20 33
1 18
33 18
3 43
44 55
6 9
4 23
21 49
55 38
48 12
7 17
45 30
11 39
40 40
27 46
40 38
52 16
4 9
34 12
9 31
41 49
16 46
34 3
32 11
22 5
52 24
41 54
16 45
44 21
15 50
42 6
38 46
16 6
8 29
1 35
28 43
41 35
18 15
55 37
53 36
23 13
27 54
45 46
4 10
7 3
47 23
30 1
27 32
31 51
4 38
6 1
48 21
6 48
24 28
9 17
32 2
1 27
9 2
10 11
29 4
36 18
54 27
15 19
47 29
23 31
23 12
49 40
17 31
30 11
33 45
16 38
19 5
2 54
1 8
17 41
54 44
9 41
21 9
19 23
8 35
23 51
49 20
43 38
6 50
46 13
34 39
10 43
46 36
33 10
35 28
19 4
28 49
27 3
55 25
19 37
34 27
8 41
41 41
52 5
4 6
21 13
29 14
37 10
20 30
49 48
39 13
26 54
5 19
33 48
9 40
29 52
9 48
20 48
27 36
10 29
52 27
49 52
3 29
34 55
4 24
1 10
30 54
37 44
40 1
11 46
52 9
45 32
50 11
52 42
30 47
11 13
44 7
3 21
43 43
21 14
51 8
7 4
24 18
17 28
44 19
36 11
11 50
23 55
11 34
21 40
45 8
26 29
51 42
19 20
46 10
31 16
4 34
40 47
54 6
47 9
37 26
37 19
4 20
28 44
12 17
3 23
14 30
8 50
46 52
13 8
20 4
24 53
28 28
53 42
36 4
43 39
3 14
9 39
5 13
54 20
5 28
48 50
8 47
50 39
35 27
36 29
50 36
14 25
36 6
50 50
19 3
21 31
23 30
3 5
26 46
8 25
45 11
44 26
34 45
21 3
47 34
10 36
51 12
39 27
39 46
23 11
30 31
34 36
31 50
32 41
17 26
38 26
53 29
54 35
55 42
26 16
50 44
26 8
11 35
11 36
5 55
35 7
2 24
26 53
20 5
1 44
7 44
23 7
26 47
3 16
5 17
42 41
48 19
16 36
24 6
2 45
49 17
45 2
24 8
18 48
25 41
29 29
33 47
42 8
42 13
23 25
17 23
24 33
28 8
7 42
46 43
27 39
1 31
10 15
2 46
5 2
40 43
1 24
25 4
20 25
11 28
32 9
36 17
10 37
13 22
53 50
23 9
32 25
12 22
45 9
54 34
3 48
26 19
30 22
44 11
38 8
45 10
18 8
54 4
26 1
1 9
53 44
25 50
23 45
50 37
11 41
37 27
25 17
21 23
48 10
24 30
2 21
38 18
22 42
50 52
11 23
47 11
40 26
37 12
43 5
14 54
51 35
28 35
46 53
15 24
5 25
10 25
20 1
48 30
36 8
29 39
17 32
55 51
35 38
55 55
22 3
47 12
15 1
20 41
2 1
33 38
45 31
32 3
13 25
54 30
23 24
37 46
31 31
55 35
47 4
2 51
17 55
33 40
36 52
33 36
19 38
33 20
55 48
39 47
3 22
50 31
9 8
18 53
31 33
4 48
21 12
14 8
28 27
25 27
51 21
24 41
29 31
44 14
8 53
3 31
4 47
6 13
27 2
49 51
7 55
51 46
49 28
24 36
34 38
54 21
11 7
21 18
37 38
51 18
21 19
12 55
51 1
33 49
1 3
7 6
43 12
8 32
12 52
54 9
51 26
11 29
16 34
36 35
19 31
38 47
48 29
5 47
43 3
16 44
43 44
54 7
8 13
44 33
17 42
5 9
14 33
25 49
32 23
38 24
33 50
4 4
45 53
37 51
37 54
40 44
34 34
29 24
6 46
23 16
42 14
35 43
7 33
35 4
33 27
34 29
15 7
9 6
45 45
5 39
21 44
30 4
17 8
23 4
54 10
49 49
16 14
53 35
18 10
19 36
1 32
52 30
47 13
47 42
19 51
35 50
50 33
49 32
12 44
49 14
15 16
35 8
32 5
19 1
48 27
25 46
40 13
49 3
41 6
19 34
31 3
3 49
19 40
17 34
28 37
38 31
2 26
39 22
40 18
42 12
13 34
47 41
41 43
26 3
3 13
13 27
2 3
16 33
32 37
50 18
22 14
41 10
21 45
7 23
51 33
41 14
9 14
1 49
42 16
1 16
16 35
5 49
25 3
27 52
54 46
12 29
9 11
34 16
26 15
3 2
44 29
19 47
18 19
2 2
38 49
22 9
17 38
54 5
48 55
31 39
30 33
27 22
28 11
10 31
10 20
45 20
30 28
15 13
47 46
6 31
41 23
26 5
42 50
43 37
21 25
53 13
30 32
3 54
55 4
39 18
51 6
16 4
51 20
33 15
4 49
34 50
15 15
4 30
27 34
47 35
4 15
13 36
32 44
48 16
13 38
6 11
50 43
41 22
5 38
38 54
30 43
9 25
30 39
8 15
34 22
29 12
41 32
24 50
24 24
22 2
16 7
42 55
37 8
24 32
22 21
54 18
26 2
10 12
39 12
55 53
50 4
28 46
34 10
48 9
18 30
2 6
36 20
4 40
39 20
21 34
2 17
