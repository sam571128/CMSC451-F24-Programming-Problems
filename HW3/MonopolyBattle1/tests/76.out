84 73 73 73 115 73 73 84 73 73 73 178 73 115 107 73 83 73 80 73 101 124 126 72 73 115 73 73 107 107 73 73 73 175 96 73 73 73 73 73 73 73 73 73 107 97 73 178 152 84 81 80 107 73 73 